### PR TITLE
bpo-29910: IDLE no longer delete character after commenting out a region

### PR DESCRIPTION
**Pull request by serhiy-storchaka**
by key shortcut.  Fixed also other potential conflicts between IDLE's and
default key bindings.

### CODE REVIEW

**Review by terryjreedy — CHANGES_REQUESTED** [review]
Review on b.p.o, tests needed

File: Misc/NEWS [hunk 1]
```diff
@@ -2152,6 +2152,10 @@ Library
 IDLE
 ----
 
+- Issue #29910: IDLE no longer delete character after commenting out a region
+  by key shortcut.  Fixed also other potential conflicts between IDLE's and
+  default key bindings.
+
```
Comment (terryjreedy) [inline Misc/NEWS]: I will move this to separate file, as now required.

Comment (terryjreedy) [inline Misc/NEWS]: This news item was misplaced anyway, under the 3.6b1 section.

**Comment by serhiy-storchaka** [comment]
Thanks for merging this Terry.

**Pull request merged**